README.md

```diff
@@ -1 +1,57 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+# Awesome-books-with-ES6
+
+# Awessom Books
+
+> This is a simple html page which is contains a dynamic form about Awessom books, books title, and the author. I have used HTML and Javascript for this project and you can use it in your local machin just by cloning this repo.
+
+
+## 🖥️ Live Demo
+
+[Awessom Books]()
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+This project built with the help of HTML and JavaScript you can see and use the cods by cloning this repo on your local machine by Git and GitHub.
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [OPTIONAL] Please it will be nice to add explicit steps on how to clone and run your app locally as this will be of help to non technical users.